sockeye/vocab.py

```diff
@@ -147,3 +150,24 @@ def reverse_vocab(vocab: Mapping) -> Dict:
     :return: A mapping from values to keys.
     """
     return {v: k for k, v in vocab.items()}
+
+
+def main():
+    params = argparse.ArgumentParser(description='CLI to build source and target vocab(s).')
+    arguments.add_build_vocab_args(params)
+    args = params.parse_args()
+
+    num_words, _ = args.num_words
```

Comment:
would it make sense to assert that both values are identical to make it explicit that for this CLI only one value should be provided?

Reply:
good idea, will do that